Source/Other/Extensions/DateFormatterExtension.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  DateFormatterExtension.swift
+//  Sequel Ace
+//
+//  Created by James on 3/11/2020.
+//  Copyright © 2020 Sequel-Ace. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
+	
+	@objc public static var mediumStyleFormatter: DateFormatter = {
+			
+		let formatter = DateFormatter()
+		formatter.formatterBehavior = Behavior.behavior10_4
+		formatter.dateStyle = Style.medium
```

Comment:
Just a nit pick (this is more just as FYI) - you can remove the `Style` and use just `formatter.dateStyle = .medium` - Swift understands it's own type

Reply:
ok ... CI fixed, will check on behaviour.